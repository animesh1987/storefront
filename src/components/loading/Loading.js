import React from 'react';
import './Loading.css';

export const Loading = () => (
    <div className="Loader"></div>
);
