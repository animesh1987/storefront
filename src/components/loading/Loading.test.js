import React from 'react';
import { create } from 'react-test-renderer';
import { Loading } from './Loading';
 
it('renders product page correctly', () => {
  const component = create(<Loading />)
    .toJSON();
  expect(component).toMatchSnapshot();
});
