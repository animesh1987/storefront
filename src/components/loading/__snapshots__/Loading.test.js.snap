// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders product page correctly 1`] = `
<div
  className="Loader"
/>
`;
